letsencrypt/futureofwork.gov.sg.conf

```diff
@@ -0,0 +1,8 @@
+server {
+    listen          443 ssl http2;
+    listen          [::]:443 ssl http2;
+    server_name     futureofwork.gov.sg;
+    ssl_certificate /etc/letsencrypt/live/futureofwork.gov.sg/fullchain.pem;
+    ssl_certificate_key     /etc/letsencrypt/live/futureofwork.gov.sg/privkey.pem;
+    return          301 https://futureofwork.gov.sg$request_uri;
+}
```

Comment:
should this be
`return 301 https://www.futureofwork.gov.sg$request_uri;`?

Reply:
Edited!